pkg/backends/calico/client.go

```diff
@@ -1073,6 +1076,14 @@ func getNodeName(nodeName string) string {
 	return strings.TrimPrefix(nodeName, perNodeConfigNamePrefix)
 }
 
+func getCommunitiesArray(communitiesMap map[string]struct{}) []string {
+	var communityValue []string
+	for k := range communitiesMap {
+		communityValue = append(communityValue, k)
+	}
+	return communityValue
```

Comment:
With the set package this would be
```
   communitiesSet.Iter(func(item interface{}) error {
     communityValue = append(communityValue, item.(string))
     return nil
    })
```